gui/wxpython/gui_core/mapdisp.py

```diff
@@ -160,17 +160,17 @@ def GetToolSwitcher(self):
 
     def SetProperty(self, name, value):
         """Sets property"""
-        if self.HasProperty("projection"):
-            self.statusbarManager.SetProperty(name, value)
+        if hasattr(self.mapWindowProperties, name):
+            setattr(self.mapWindowProperties, name, value)
         else:
-            self.mapWindowProperties.useDefinedProjection = value
+            self.statusbarManager.SetProperty(name, value)
 
     def GetProperty(self, name):
         """Returns property"""
-        if self.HasProperty("projection"):
-            return self.statusbarManager.GetProperty(name)
+        if hasattr(self.mapWindowProperties, name):
+            getattr(self.mapWindowProperties, name)
```

Comment:
```suggestion
            return getattr(self.mapWindowProperties, name)
```

Reply:
Nice! I have just realized the error :-) In the same time you post it. :-D